src/pages/admin/pages/faq/FaqPage.vue

```diff
@@ -3,16 +3,18 @@
     <va-card class="mb-8">
```

Comment:
There are 2 sources of truth right now:
* design
* vuestic-ui

If there is a conflict between two - we have to discuss (i.e. vuestic-ui is hard to customize or some customization is needed at all).

Vuestic-admin current state is not source of truth and should be ignored completely. (both visuals and structure)


<img width="1194" alt="image" src="https://github.com/epicmaxco/vuestic-admin/assets/5394573/6ed25e38-2116-46ce-b8a2-609a37a1e39c">